### DISCUSSION TITLE
Is it recommended to specify tracks for all reads?

### DISCUSSION THREAD

**Discussion by wearwolf** [keirf/greaseweazle#345] [Q&A] [answered]
I'm mainly looking to archive disks for the IBM PC and I'm curious about the recommended way to handle the number of tracks to read.

The default for gw read is c=0-81 but the example for a 40 track drive is c=0-39.

So I'm not sure if I should be using c=0-79 for 80 track disks or c=0-41 for 40 track disks

Is there anything in those extra tracks that might be useful for archival purposes or for accurately reproducing the disk? 

Thanks

**Comment by keirf** [comment]
Generally no. There can be duplicator information contained there but it's not part of the main disk data and is of no practical value.

**Reply by wearwolf** [reply]
So do people typically read 80 track disks with c=0-79 and 40 track disks with c=0-39?

**Reply by keirf** [reply, marked as answer]
Yes at least if they do a read and decode. People doing raw dumps might dump an extra cylinder or two but I would only really bother if there is reason to suspect extra data there. Eg some Atari ST and C64 disks get crammed with extra tracks. And some Amiga games use extra tracks.

**Reply by wearwolf** [reply]
Good to know, thank you
